core/templates/dev/head/components/ProfileLinkImageDirective.js

```diff
@@ -32,18 +32,21 @@ oppia.directive('profileLinkImage', [function() {
         return GLOBALS.SYSTEM_USERNAMES.indexOf(username) == -1;
       };
 
-      $scope.profileImageUrl = (
-        '/preferenceshandler/profile_picture_by_username/' + $scope.username());
-      $scope.profilePicture = DEFAULT_PROFILE_IMAGE_PATH;
+      if ($scope.username() !== 'Community Owned') {
```

Comment:
I don't think you should use this directive for displaying the 'community owned' image; 'Community Owned' does not represent a person's profile.

One thing you could do is to create a new directive called circularImage or similar. It takes in a picture and an optional link, and displays a circle with the picture (and, if the link is specified, clicking on the circle takes the user to the link's target). Then the stuff in explorationSummaryTile can just do an ng-repeat on directives of this form. You could optionally also change the html here to refer to that directive -- this both removes duplication and ensures the conceptual descriptions of the directives are accurate.